.github/workflows/jit.yml

```
@@ -5,11 +5,17 @@ on:
       - '**jit**'
       - 'Python/bytecodes.c'
       - 'Python/optimizer*.c'
+      - '!Python/perf_jit_trampoline.c'
+      - '!**/*.md'
+      - '!**/*.ini'
   push:
     paths:
       - '**jit**'
       - 'Python/bytecodes.c'
       - 'Python/optimizer*.c'
+      - '!Python/perf_jit_trampoline.c'
+      - '!**/*/README.md'
+      - '!**/*/mypy.ini'
```

These should be the same as above, right? Or am I missing something?

```suggestion
      - '!**/*.md'
      - '!**/*.ini'
```

---

Oops, I think that one's on me!